arcgis-ios-sdk-samples/Content Display Logic/Controllers/SampleInfoViewController.swift

```diff
@@ -54,13 +54,15 @@ class SampleInfoViewController: UIViewController {
     
     func displayHTML(_ readmeContent: String) {
         let cssPath = Bundle.main.path(forResource: "style", ofType: "css") ?? ""
+        let darkmodePath = Bundle.main.path(forResource: "darkmode", ofType: "css") ?? ""
         let string = """
             <!doctype html>
             <html>
             <head>
                 <link rel="stylesheet" href="\(cssPath)">
                 <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.2/css/foundation.min.css">
                 <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+                <link rel="stylesheet" href="\(darkmodePath)">
```

Comment:
Add an additional css to only adapt a few colors to dark mode automatically, to avoid changing the original css style.